### DISCUSSION TITLE
Remove mesh values

### DISCUSSION THREAD

**Discussion by DeNohvo** [Jyers/Marlin#185] [Q&A] [answered]
Once we set the manual mesh is there a way to delete or clear those values from the EEPROM without resetting all other settings?

**Comment by Jyers** [comment, marked as answer]
Not that I am aware of, you can either disable the mesh with gcode or remake it to overwrite the old values.

**Reply by DeNohvo** [reply]
M420 S0 to disable?

If i'm not mistaken your firmware pulls the stored values after each print anyways correct?

**Reply by Schnabulation** [reply]
That is Marlins job, I believe. Once a mesh is stored and activated by `M420 S1` it will ger applied to any movements of the Z-axis.

**Reply by DeNohvo** [reply]
right but if i remove M420 S1 from my slicer gcode and power off/on the printer am i correct to assume the mesh values will not be  used for that print? I read in a other post the mesh values are pulled by pre complied firmware by default.  

Thats why i was wondering if M420 S0 is used to clear the stored values.

**Reply by Jyers** [reply]
M420 S0 will not clear stored values, but it will disable the mesh, I believe the mesh is also disabled my default when the printer is power cycled.

**Reply by Schnabulation** [reply]
Yes it is, I just checked.

**Reply by xantrk** [reply]
> Yes it is, I just checked.

Will M420 S1 restore the mesh after power cycle?

**Reply by Schnabulation** [reply]
Yes. As long as the mesh is saved to EEPROM.
